libcontainer/specconv/spec_linux.go

```diff
@@ -180,20 +180,11 @@ func CreateLibcontainerConfig(opts *CreateOpts) (*configs.Config, error) {
 	}
 
 	exists := false
-	if config.RootPropagation, exists = mountPropagationMapping[spec.Linux.RootfsPropagation]; !exists {
-		return nil, fmt.Errorf("rootfsPropagation=%v is not supported", spec.Linux.RootfsPropagation)
+	linuxSpecific := false
+	if spec.Linux != nil {
+		linuxSpecific = true
 	}
```

Comment:
This variable is not necessary since you only use it once. As an aside, the nice way of writing this would be

```go
linuxSpecific := spec.Linux != nil
```